www/front_src/src/Resources/Details/Header.tsx

```diff
@@ -194,10 +208,27 @@ const Header = ({ details, onSelectParent }: Props): JSX.Element => {
           </div>
         )}
       </div>
-      <ShortcutsTooltip
-        data-testid={labelShortcuts}
-        resourceUris={resourceUris}
-      />
+
+      <IconButton
+        ariaLabel={t(labelViewLogs)}
+        data-testid={labelViewLogs}
+        size="small"
+        title={t(labelViewLogs)}
+        onClick={(): void => navigateToResourceUris('logs')}
```

Comment:
Transform `navigateToResourceUris` to high order function so it will be easier to use
```suggestion
        onClick={navigateToResourceUris('logs')}
```